packages/fxa-shared/l10n/branding.ftl

```diff
@@ -0,0 +1,42 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### Localization for fxa
+
+## Brand and product terms to be used globally
+
+# Note: Terms should not be transliterated or translated
+# Declension should be avoided
+
+# Firefox and Mozilla Brand
+-brand-name-mozilla = Mozilla
```

Comment:
> The draft file proposes one format: '-brand-name' and '-product-name' based on the format used in payments-server. Could be simplified to '-brand'. Does the team have a preference?

Personally, I prefer dropping `-name` for simplicity, but I would go with whatever we're using the most. It looks like we have a mix of `-brand-name-mozilla` and `-brand-mozilla`, but less references to `-brand-name-mozilla` thankfully. If overall, keeping `-name` means less updates on the l10n side, I'm fine with going with that though.

Reply:
I'd gone with `-name` because that format was used for all of the terms in the `main.ftl` file, but I'd personally favour going for the shorter format `-brand-mozilla`, _especially_ if it results in fewer updates on the l10n side.